### PR TITLE
[READY] Assistant Operatives

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

This PR adds a new gamemode, Assistant Operatives! A variant of nuclear, this gamemode features Assistants hired by the Syndicate trying to nuke the station. 

Features include:

- Unique armor and hardsuits
- Unique weapons
- Unique character names and items

This is essentially Nuclear but a variant, just like Clown Ops.

## Why It's Good For The Game

Have you ever been pissed off at NanoTrasen? Do you just want to nuke Security for being shitsec? Well, now you can! With up to 10 boys in grey, you can take advanced makeshift weapons and toolboxes to bash their heads in and blow them sky high!

This game mode will add more variety to the game mode roster which is something that has been wanted for a while now.

More images will come once code is sufficiently written enough to have them in game.

## Changelog
:cl:
add: Added initial code
/:cl:

### CODE REVIEW

File: code/game/gamemodes/assistant_ops/assistant_ops.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	mask = /obj/item/clothing/mask/gas
+	gloves = /obj/item/clothing/gloves/color/yellow
+	back = /obj/item/storage/backpack
+	ears = /obj/item/radio/headset/syndicate/alt
+	l_pocket = /obj/item/pinpointer/nuke/syndicate
+	r_pocket = /obj/item/storage/toolbox/greytide
+	id = /obj/item/card/id/syndicate
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,
+		/obj/item/kitchen/knife/combat/survival,
+		/datum/design/jawsoflife)
```
Comment (Fikou) [inline code/game/gamemodes/assistant_ops/assistant_ops.dm]: lol

**Comment by ghost** [comment]
All major features have been added. It has been tested on my local machine and it works.

File: code/game/gamemodes/assistant_ops/assistant_ops.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (LemonInTheDark) [inline code/game/gamemodes/assistant_ops/assistant_ops.dm]: Lol, but also does this work? You are trying to give them a datum, which is a thing that by it's nature does not exist in game

File: code/game/gamemodes/assistant_ops/scrap_bomb.dm [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+		if(!H.w_uniform || H.dropItemToGround(H.w_uniform))
+			C = new /obj/item/clothing/under/color/grey(H)
+			H.equip_to_slot_or_del(C, ITEM_SLOT_ICLOTHING)
+		if(!H.shoes || H.dropItemToGround(H.shoes))
+			C = new /obj/item/clothing/shoes/sneakers/black(H)
+			H.equip_to_slot_or_del(C, ITEM_SLOT_FEET)
+
+		if(!H.wear_mask || H.dropItemToGround(H.wear_mask))
+			C = new /obj/item/clothing/mask/gas(H)
+			H.equip_to_slot_or_del(C, ITEM_SLOT_MASK)
```
Comment (LemonInTheDark) [inline code/game/gamemodes/assistant_ops/scrap_bomb.dm:35]: Quality

Comment (LemonInTheDark) [reply code/game/gamemodes/assistant_ops/scrap_bomb.dm:35]: this is tested I assume?

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 4]
```diff
@@ -505,6 +505,31 @@
...
+//                                          //
+//             ASSISTANT OPS                //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/nuclear/assistant_ops
+	name = "Assistant Ops"
+	antag_datum = /datum/antagonist/nukeop/assistantop
+	antag_leader_datum = /datum/antagonist/nukeop/leader/assistantop
+	requirements = list(101,101,101,101,101,101,101,101,101,101)
```
Comment (LemonInTheDark) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm]: You aware of what these values mean?

**Review by LemonInTheDark — COMMENTED** [review]
Top tier

File: code/game/gamemodes/assistant_ops/scrap_bomb.dm (hunk 3 above)
Comment (ghost) [reply code/game/gamemodes/assistant_ops/scrap_bomb.dm:35]: Havent been able to spawn as one due to needing more than 1 person. Ill try with some others later

File: code/game/gamemodes/assistant_ops/assistant_ops.dm (hunk 2 above)
Comment (ghost) [reply code/game/gamemodes/assistant_ops/assistant_ops.dm]: Yea that was a mistake by me while I was searching for the jaws of life, mustve ctrl c'ed too quickly

File: code/game/gamemodes/clown_ops/bananium_bomb.dm [hunk 5]
```diff
@@ -17,7 +17,7 @@
 		if(NUKE_ON_EXPLODING)
 			icon_state = "bananiumbomb_exploding"
 
-/obj/machinery/nuclearbomb/syndicate/bananium/get_cinematic_type(off_station)
+/obj/machinery/nuclearbomb/syndicate/scrap/get_cinematic_type(off_station)
```
Comment (Wym0n) [inline code/game/gamemodes/clown_ops/bananium_bomb.dm]: Did you mean to change this?

Comment (ghost) [reply code/game/gamemodes/clown_ops/bananium_bomb.dm]: it was a mistake

**Comment by LemonInTheDark** [comment]
Here ya go :) https://github.com/PapaSejd/Fulpstation/pull/1

**Pull request closed without merging**